.gemini/settings.json

```diff
@@ -2,12 +2,12 @@
 {
     "mcpServers": {
         "github": {
-            "command": "/server/github-mcp-server",
```

Comment:
@barney-s I originally had this set based on the sandbox path. Sandbox is no longer functional but not sure if this affects the repo-agent.